spark/src/test/java/org/apache/iceberg/actions/TestRewriteDataFilesAction.java

```diff
@@ -347,15 +348,19 @@ public void testRewriteDataFilesForLargeFile() throws AnalysisException {
 
     Actions actions = Actions.forTable(table);
 
-    long targetSizeInBytes = maxSizeFile.fileSizeInBytes() - 10;
+    long targetSizeInBytes = maxSizeFile.fileSizeInBytes() / 2;
     RewriteDataFilesActionResult result = actions
         .rewriteDataFiles()
         .targetSizeInBytes(targetSizeInBytes)
         .splitOpenFileCost(1)
         .execute();
 
-    Assert.assertEquals("Action should delete 4 data files", 4, result.deletedDataFiles().size());
-    Assert.assertEquals("Action should add 2 data files", 2, result.addedDataFiles().size());
+    // 1 big datafile and 2 small datafiles
+    Assert.assertEquals("Action should delete 3 data files", 3, result.deletedDataFiles().size());
```

Comment:
I actually noticed this when I did the new RewriteDatafile api because this test did fail because the output here isn't quiet correct.